hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/snapshot/SnapshotDiffResponse.java

```diff
@@ -0,0 +1,85 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.hadoop.ozone.snapshot;
+
+import org.apache.hadoop.ozone.protocol.proto.OzoneManagerProtocolProtos.SnapshotDiffResponse.JobStatusProto;
+
+/**
+ * POJO for Snapshot Diff Response.
+ */
+public class SnapshotDiffResponse {
+
+  /**
+   * Snapshot diff job status enum.
+   */
+  public enum JobStatus {
+    QUEUED,
+    IN_PROGRESS,
```

Comment:
IN_PROGRESS, can be further broken down into multiple stages , DAG walk, SST file level Diff, ObjectOID based diff calculation and then "DONE" etc.

Reply:
Should we use sub categories for that?

Reply:
I think using one category is enough. This change can be a subsequent PR too. Please file a Jira to track it in a separate PR.

Reply:
Created jira: https://issues.apache.org/jira/browse/HDDS-8007.